unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs

```diff
@@ -135,9 +142,67 @@ private void EmoteSell(string collectibleId)
             var ownedCollectible = ownedNftCollectionsL1.FirstOrDefault(nft => nft.urn == collectibleId) ??
                                    ownedNftCollectionsL2.FirstOrDefault(nft => nft.urn == collectibleId);
 
-            WebInterface.OpenURL(ownedCollectible != null ?
-                URL_SELL_SPECIFIC_COLLECTIBLE.Replace("{collectionId}", ownedCollectible.collectionId).Replace("{tokenId}", ownedCollectible.tokenId) :
-                URL_SELL_COLLECTIBLE_GENERIC);
+            WebInterface.OpenURL(ownedCollectible != null ? URL_SELL_SPECIFIC_COLLECTIBLE.Replace("{collectionId}", ownedCollectible.collectionId).Replace("{tokenId}", ownedCollectible.tokenId) : URL_SELL_COLLECTIBLE_GENERIC);
+        }
+
+        private async UniTask<IEnumerable<WearableItem>> FetchCustomEmoteItems(CancellationToken cancellationToken)
```

Comment:
You don't need to construct a collection as you just add it to another one. You can pass the latter directly to the method

Reply:
Done